src/vivarium_public_health/risks/base_risk.py

```diff
@@ -158,6 +156,9 @@ def get_exposure_distribution(self) -> SimulationDistribution:
     # Setup methods #
     #################
 
+    def build_lookup_tables(self, builder: "Builder") -> None:
```

Comment:
For the other reviewers I found this confusing at first - this is overwriting the component method it inherits to do nothing and then the subcomponent Risk creates is overwriting this method again.